hiddenlayer/sdk/services/model_scan.py

```diff
@@ -351,11 +365,17 @@ def scan_huggingface_model(
             wait_for_results=wait_for_results,
         )
 
-    def get_scan_results(self, *, model_name: str) -> ScanResults:
+    def get_scan_results(
+        self,
+        *,
+        model_name: str,
+        model_version: Optional[int] = None,
+    ) -> ScanResults:
         """
         Get results from a model scan.
 
         :param model_name: Name of the model.
+        :param model_version: Version of the model.
 
         :returns: Scan results.
```

Comment:
Realizing that this is simpler than I'm thinking because we're calling scan_file or scan_folder synchronously for the user, even though inside the function it's async. Inside the function if the user called us with a version then we have it in our mitts to get the right results.